### DISCUSSION TITLE
Welcome to trombi-srs-2021 Discussions!

### DISCUSSION THREAD

**Discussion by dehboris** [dehboris/trombi-srs-2021#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ddereck** [comment]
Bonjour Mr, j'ai une préoccupation, je suis au niveau de la partie 3. J'aimerais savoir si le travail à ce niveau doit se faire dans le terminal de l'éditeur ou au niveau du site ?

J'ai essayé au niveau de l'éditeur avec la commande "git remote add upstream https://github.com/dehboris/trombi-srs-2021.git " et ensuite avec "git fetch upstream" , le résultat montre que la branche upstream/main a été créée

>  [new branch] main -> upstream/main

mais` on ne la voit pas sur la liste des branches !

**Reply by dehboris** [reply]
Le fait que tu es eu ce message montre que la branche a été créer. 

Tu ne le vois pas parce que ce n'est pas une branche **local**. 

Tu as la possibilité de te positionner sur cette branche en faisant `git checkout upstream/main` mais le but est le fusionner cette branche avec ta branche `main` en local.

**Comment by ConstanciG** [comment]
svp monsieur j'ai eu un probleme de conflit et j'ai choisi l'option 3 comme on l'a fait en classe mais le problemme de conflit continue toujours parcequ'au niveau de la partie 4 ça me dit qu'il y a conflit
